en/developer_guide/form_type.rst

```diff
@@ -0,0 +1,311 @@
+| ``GroupType``                     | ``oo_group``                      |Group Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``KeywordType``                   | ``oo_keyword``                    |Keyword Form           |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``NodeType``                      | ``oo_node``                       |node Form              |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``OrchestraChoiceType``           | ``choice``                        |                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``RedirectionType``               | ``oo_redirection``                |redirection Form       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``RoleType``                      | ``oo_role``                       |roles Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``SiteAliasType``                 | ``oo_site_alias``                 |Add aliase Field       |
+|                                   |                                   |on SiteType            |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``SiteType``                      | ``oo_site``                       |Site Form              |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``StatusType``                    | ``oo_status``                     |status Form            |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TemplateType``                  | ``oo_template``                   |template Form          |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``ThemeType``                     | ``oo_theme``                      |theme Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TinymceType``                   | ``oo_tinymce``                    |                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TranslatedValueCollectionType`` | ``oo_translated_value_collection``|                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TranslatedValueType``           | ``oo_translated_value``           |                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``VideoType``                     | ``oo_video``                      |                       |
```

Comment:
This form type is displayed as a text line where the user put a video url from major video platforms such as youtube or dailymotion.